presto-main/src/main/java/com/facebook/presto/type/DecimalInequalityOperators.java

```diff
@@ -174,6 +199,21 @@ public static boolean opShortShortShortRescale(long a, long b, long aRescale, lo
     }
 
     @UsedByGeneratedCode
+    public static boolean opShortShortShortRescaleNullable(
+            long a, boolean aNull,
+            long b, boolean bNull,
+            long aRescale, long bRescale, MethodHandle getResultMethodHandle)
+    {
+        if (aNull != bNull) {
+            return true;
+        }
+        if (aNull && bNull) {
```

Comment:
You forgot to change these `aNull && bNull` to just `aNull`. If IntelliJ is not giving you warnings, you probably want to configure your IntelliJ so that it does.